.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Thu Sep 20 18:05:58 CDT 2018
```

Comment:
Please delete this `.gradle` folder as this looks redundant

Reply:
Its done . Kindly have a  check again. I have checked in the code so should be visible in the PR